keyboards/xd002/keymaps/multilayer_rgb/config.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2020 elmo-space<mail@elmo.space>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*enable layers*/
+#undef NO_ACTION_LAYER
```

Comment:
You may also want to reduce the size of the layer state bit mask, to save space: 
```suggestion
#undef NO_ACTION_LAYER

#define LAYER_STATE_8BIT
```

Reply:
Already set at the keyboard level
<https://github.com/qmk/qmk_firmware/blob/13206b2f87b1bce87d58cdfdddb1bcc7b58e4a24/keyboards/xd002/config.h#L45>